code/__DEFINES/subsystems.dm

```diff
@@ -331,3 +331,7 @@
 
 /// The timer key used to know how long subsystem initialization takes
 #define SS_INIT_TIMER_KEY "ss_init"
+
+// Vote subsystem counting methods
+#define VOTE_COUNT_METHOD_FPTP 1
+#define VOTE_COUNT_METHOD_AV 2
```

Comment:
You need to document what these are so people don't have to google FPTP and AV

Reply:
Ah yeah, got it.